features/rules/ALB/ALB010_Alignment-nesting-referents.feature

```diff
@@ -1,15 +1,18 @@
 @industry-practice
 @ALB
-@version1
+@version2
 @E00100
 Feature: ALB010 - Alignment Nesting Referents
-The rule verifies that each alignment nests at least one IfcReferent, such as stations or mileage points. 
+The rule verifies that each horizontal alignment nests at least one IfcReferent, such as stations or mileage points.
 These can be used as semantic entities holding information about locations along the alignment. 
 IfcReferent is associated to IfcAlignment via the IfcRelNests relationship.
 
-  Scenario: Agreement on each IfcAlignment nesting at least one IfcReferent
+  Scenario: Agreement on each IfcAlignment nesting at least one IfcReferent when that alignment nests an IfcAlignmentHorizontal
 
       Given A model with Schema 'IFC4.3'
-      Given an .IfcAlignment.
-      
-      Then A relationship IfcRelNests must exist from IfcAlignment to IfcReferent
\ No newline at end of file
+      Given an .IfcAlignmentHorizontal.
+      Given its attribute .Nests.
+      Given its attribute .RelatingObject.
+
+      Then a relationship .IfcRelNests. must exist from .IfcAlignment. to .IfcReferent.
```

Comment:
One refactoring-step further, nice :)

Reply:
Just did an attempt to add them, like this?
https://github.com/buildingSMART/ifc-gherkin-rules/pull/378/commits/76e347543185a4d3779bc2f22c9d6903cf66b208